### PR TITLE
Consensus seg update for oncoplot

### PR DESCRIPTION
**Pull request by kgaonkar6**
#### What scientific question is your analysis addressing?
We are adding CNV calls from consensus seg files for autosomes , X and Y chromosomes as inputs for oncoplots. 

#### What was your approach?
Update the input files to : 
```
consensus_seg_autosomes_cnv_file=${focal_directory}/consensus_seg_annotated_cn_autosomes.tsv.gz
consensus_seg_cnv_xy_cnv_file=${focal_directory}/consensus_seg_annotated_cn_x_and_y.tsv.gz
```

#### What GitHub issue does your pull request address?
#1087

#### Which areas should receive a particularly close look?
The figures need careful review , the only update in consensus seg files and we are currently coding loss as "Del" and amplification and gain as "Amp" to match the maftools requirements as in #1009 

#### Is there anything that you want to discuss further?
Na

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes

#### What types of results are included (e.g., table, figure)?
figures

#### What is your summary of the results?
ATRX calls are now added with use of X and Y chromosome calls and all CNV overlapping gene locations are being plotted. 

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/oncoprint-landscape/01-map-to-sample_id.R [hunk 1]
```diff
@@ -283,11 +295,10 @@ cnv_df <- cnv_df %>%
   inner_join(select(histologies_df,
                     Kids_First_Biospecimen_ID,
                     sample_id),
-             by = "Kids_First_Biospecimen_ID") %>%
-  filter(status != "uncallable") %>%
+             by = c("biospecimen_id"="Kids_First_Biospecimen_ID")) %>%
   mutate(Tumor_Sample_Barcode =  sample_id) %>%
   rename(Variant_Classification = status,
```
Comment (jharenza) [inline analyses/oncoprint-landscape/01-map-to-sample_id.R:300]: per [this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1005#issuecomment-836671602), we need to separate out gains and amplifications here and only carry through amps to the oncopolot - can you update this?

**Review by jharenza — CHANGES_REQUESTED** [review]
Thanks @kgaonkar6 for these changes. It looks like you just did a basic swap of the consensus seg in place of the focal CN file, but we also need to code the amps appropriately (I think we had used CN*2(?) throughout this repository). Can you make that adjustment with this PR? Thanks!

File: analyses/oncoprint-landscape/01-map-to-sample_id.R (hunk 1 above)
Comment (jaclyn-taroni) [reply analyses/oncoprint-landscape/01-map-to-sample_id.R:300]: To give a little more context (and to interject with my opinion about _how_ to do this)... 

Currently we recode this in `02-plot-oncoprint.R`: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/c0744576ee80914636443b01012168efc2da22a9/analyses/oncoprint-landscape/02-plot-oncoprint.R#L149

I would remove that from `02-plot-oncoprint.R` and move it into this script (`01-map-to-sample_id.R`), where you mutate `Variant_Classification`:

```R
dplyr::case_when(
  Variant_Classification == "loss" ~ "Del",
  Variant_Classification  == "amplification" ~ "Amp",
  TRUE ~ as.character(Variant_Classification)
)
```

And then filter with `Variant_Classification %in%  c("Del", "Amp")`

Comment (kgaonkar6) [reply analyses/oncoprint-landscape/01-map-to-sample_id.R:300]: Updated now

**Comment by jaclyn-taroni** [comment]
Just re-requested @jharenza, I'm going to wait to review #1092 until this gets another look

**Comment by jharenza** [comment]
Thanks, I took a brief look yesterday, but think we also need a "deep deletion filter", as hemizygous deletions aren't usually shown in the oncoprints, but I need to think and read about this a little more before proposing a threshold.

**Comment by jaclyn-taroni** [comment]
> Thanks, I took a brief look yesterday, but think we also need a "deep deletion filter", as hemizygous deletions aren't usually shown in the oncoprints, but I need to think and read about this a little more before proposing a threshold.

Is that filter necessary for this PR or can be done later, in a separate pull request, once you have had a chance to read and digest some material?

**Comment by jharenza** [comment]
> > Thanks, I took a brief look yesterday, but think we also need a "deep deletion filter", as hemizygous deletions aren't usually shown in the oncoprints, but I need to think and read about this a little more before proposing a threshold.
> 
> 
> 
> Is that filter necessary for this PR or can it be done later, in a separate pull request, once you have had a chance to read and digest some material?

We can do later, thanks!

**Review by jaclyn-taroni — APPROVED** [review]

**Pull request merged**